models/issue_label.go

```diff
@@ -126,6 +153,8 @@ func (label *Label) ForegroundColor() template.CSS {
 	return template.CSS("#000")
 }
 
+// LABEL
```

Comment:
reason for this comment?

Reply:
I wanted to separate the sections, I meant to turn them into the large letter comments like other sections but I forgot, will fix !

Reply:
fixed : )